### PR TITLE
Radiomics metadata sheet

### PR DESCRIPTION
**Pull request by ggabernet**
Hi,

I created a first example of the json schema for the radiomics metadata sheet. The example is for the first column "Filenames", in the metadata sheet:

https://docs.google.com/spreadsheets/d/1FfHAsUOmb5BI6LnrIK6bAkRKq-o7SJ0xbIRfmTKd0P4/edit#gid=1148992430

The `OpenBIS_instance` and `OpenBIS_property` indicate what would be the related OpenBIS field where this information should be stored.

@sven1103 @wow-such-code @jenniferboedker could you review that I am doing things properly here? I used the metastats model as inspiration.

### CODE REVIEW

File: metadata-sheets-schema/radiomics-metadatasheet-schema.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                    "type": "string"
+                },
+                "value": {
+                    "type": "string"
+                }
+            },
+            "required": ["label","value"],
+            "OpenBIS_intance": "Q_BMI_IMAGING_DATA",
+            "OpenBIS_property": "Q_FILENAME"
+        }
```
Comment (jenniferboedker) [inline metadata-sheets-schema/radiomics-metadatasheet-schema.json]: I am not sure if you want a definition here. If you look into https://github.com/qbicsoftware/metastats-object-model/blob/master/model.schema.json I used the definition to define how a condition looks like (they are more complex objects) and used the definition later in the schema. 
With  "type": "object", "properties": { ... } I list all the fields that are part of the schema like "Filenames" in your case. I think you mixed up the definitions part with the actual definition of the schema. You can look check this  https://json-schema.org/understanding-json-schema/ to learn more about how to create a schema. We can also meet to talk about it if you want.

Comment (ggabernet) [reply metadata-sheets-schema/radiomics-metadatasheet-schema.json]: oh thanks @jenniferboedker for your feedback. You're totally right, I oversaw the first entry was different. I fix this and ask you again :)

File: metadata-sheets-schema/radiomics-metadatasheet-schema.json [hunk 2]
```diff
@@ -3,20 +3,19 @@
...
-                },
-                "value": {
-                    "type": "string"
-                }
-            },
-            "required": ["label","value"],
-            "OpenBIS_intance": "Q_BMI_IMAGING_DATA",
+            "description": "File name / directory name of the tomogram entry.",
+            "type": "string",
+            "OpenBIS_instance": "Q_BMI_IMAGING_DATA",
```
Comment (jenniferboedker) [inline metadata-sheets-schema/radiomics-metadatasheet-schema.json]: I am not sure if you can just state these OpenBIS_* Terms. I think you need to define them on the top. Like "definitions":{ 
"OpenBIS_instance":{ 
"type":"string"},
"OpenBIS_property":{
"type":"string"}
}
maybe we should ask @sven1103 if I am right

Comment (jenniferboedker) [reply metadata-sheets-schema/radiomics-metadatasheet-schema.json]: And after the definition you can state them as you did

**Review by jenniferboedker — COMMENTED** [review]
I think its right now :) I used https://www.jsonschemavalidator.net/ to check the schema (in case you didn't know it)

**Pull request merged**